packages/react/src/visual-checkbox/index.tsx

```diff
@@ -1,4 +1,18 @@
 import { CdsInternalVisualCheckbox as VisualCheckbox } from '@cds/core/internal-components/visual-checkbox';
-import { createComponent } from '../converter/react-wrapper.js';
+import '@cds/core/internal-components/visual-checkbox/register';
+import { createComponent } from '@lit-labs/react';
+import * as React from 'react';
 
-export const CdsInternalVisualCheckbox = createComponent('cds-internal-visual-checkbox', VisualCheckbox);
+export const CdsInternalVisualCheckbox = createComponent(React, 'cds-internal-visual-checkbox', VisualCheckbox);
+
+/**
+ * Setting displayName on the prototype is a workaround for now.
+ * See: https://github.com/lit/lit/issues/2154
+ * Follow: https://github.com/lit/lit/pull/2155
+ */
+CdsInternalVisualCheckbox.displayName = 'CdsInternalVisualCheckbox';
+
+// Setting react version has to be done at component level since we now consume the @lit-labs/react package
+if ((window as any).CDS && !(window as any).CDS._react.version) {
+  (window as any).CDS._react.version = React.version;
+}
```

Comment:
@bbogdanov 's suggestion to make a utility function is a good one. if we have even a minor change to this setup, it will result in a huge number of files needing manual changes.

I wonder if we could do something like this....

```
export const CdsInternalVisualCheckbox = createReactWrapper('cds-internal-visual-checkbox', VisualCheckbox);
```

...and then inside that utility function we have...

```
import { kebabCaseToPascalCase } from '@cds/core/internal';
import { createComponent } from '@lit-labs/react';
import * as React from 'react';

export function createReactWrapper(tagName: string, cdsComonent: any) {
    const reactComponent = createComponent(React, tagName, cdsComponent);
    reactComponent.displayName = kebabCaseToPascalCase(tagName);

    if (window && (window as any)?.CDS?._react && !(window as any).CDS._react.version) {
        (window as any).CDS._react.version = React.version;
    }
} 
```

The end result doesn't dramatically shift what we had with the original converter but I think this or something in this ballpark will make the whole thing more maintainable in the long term. One change can impact 50+ files. As opposed to changing 50+ files by hand.

```

Reply:
I spent some time trying to make the `createReactWrapper` function. There are typing issue with Reacts types that complicate it a lot and I couldn't get it to work. I can add a util for the React.version check.

This change is for the v6 work so I'm ok not putting it in until after we make the v5 branch. It's not a breaking change.

Reply:
Will stay with the prototype approach as discussed. The correct fix here is in the createComponent function but this is a workaround for React dev tools until then.